2-mandatory/2-currency-code-lookup.js

```diff
@@ -19,6 +19,12 @@ const COUNTRY_CURRENCY_CODES = [
 
 function createLookup(countryCurrencyCodes) {
   // write code here
+  let countries = {};
+  for (let country in countryCurrencyCodes) {
+    countries[countryCurrencyCodes[country][0]] =
+      countryCurrencyCodes[country][1];
+  }
+  return countries;
```

Comment:
Your code works. How about make it less complicated by using for-of loop?
function createLookup(countryCurrencyCodes) {
  // write code here
  let countries = {};
  for (let country of countryCurrencyCodes) {
    countries[country[0]] = country[1]
  }
  return countries;
}